build-tools/packages/build-cli/src/flags.ts

```diff
@@ -167,7 +167,7 @@ export const checkFlags = {
     }),
     policyCheck: Flags.boolean({
         allowNo: true,
-        default: true,
+        default: true, // This value isn't used directly; the default is based on the branch. See comment in run method.
```

Comment:
Would `false` be a better fake value here since it defaults to false everywhere except main?

Reply:
Maybe, but since it doesn't matter I'm going to leave it.